base-widget/src/Widget.jsx

```diff
@@ -101,14 +102,36 @@ export default class Widget extends Component {
         return '';
     }
 
+    /**
+     * Get global widget state.
+     *
+     * @param {string} key Key
+     * @returns {{}}
+     */
+    getGlobalState(key) {
+        const states = Widget._getStateObject();
+        return Object.prototype.hasOwnProperty.call(states, key) ? states[key] : { };
```

Comment:
Can't we simple `return states[key] || {}`

Reply:
+1